drivers/pwm/Kconfig

```diff
@@ -21,6 +21,12 @@ config PWM_SHELL
 	help
 	  Enable the PWM related shell commands.
 
+config PWM_CAPTURE_API
```

Comment:
The `_API` suffix seems weird, can we just call this `PWM_CAPTURE`?

Reply:
I'd like to, but I used `_API` to match our other, conditional APIs (`FLASH_JESD216_API`, `UART_ASYNC_API`, `LEGACY_TIMEOUT_API`, `POSIX_API`, ...).

Reply:
Agree with @MaureenHelm's point, we should instead move towards removing the `_API` suffix wherever it is not strictly necessary.

Reply:
I have changed it to `CONFIG_PWM_CAPTURE` instead.